compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/assign_packet_ids.mlir

```diff
@@ -212,3 +212,55 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
     return
   }
 }
+
+
+// -----
+
+// Test that control packets should take priority in the ID assignment.
```

Comment:
Maybe add that this can be seen in the checks by those flow ops getting `packet_id == 0` assigned?